### PR TITLE
Fixed a typo

### PR DESCRIPTION
**Pull request by twlz0ne**

### CODE REVIEW

**Comment by tee3** [comment]
@twlz0ne Can you resolve the conflicts caused by merging an earlier pull request?

**Comment by twlz0ne** [comment]
@tee3 Resolved

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ See `lsp-clangd-executable` to customize the path to clangd.
   ((c-mode . lsp-clangd-c-enable)
    (c++-mode . lsp-clangd-c++-enable)
    (objc-mode . lsp-clangd-objc-enable)))
-```
+
```
Comment (tee3) [inline README.md]: This should not have been removed.

**Comment by twlz0ne** [comment]
@tee3 I've re-pushed this branch, seems clean now.

**Pull request merged**